<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>721</width>
      <height>501</height>
     </rect>
    </property>
    <widget class="QWidget" name="page_sisaan">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>321</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Pankkisimulaattori</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSisaan">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>270</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Sisään</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_menu">
     <widget class="QPushButton" name="btnSaldo">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Saldo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOtto">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Otto</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTilitapahtumat">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Tilitapahtumat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLuottorajanNosto">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>260</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Luottorajan Nosto</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPoistu_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>380</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Poistu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_valitseKortti">
     <widget class="QPushButton" name="btnDebit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Debit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCredit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>190</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Credit</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblValitse_Kortti">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Valitse kortti</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_saldo">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Saldo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTakaisin_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>370</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Takaisin</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_tilitapahtumat">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Tilitapahtumat</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTakaisin_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>370</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Takaisin</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_otto">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Otto</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTakaisin_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>370</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Takaisin</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_nostaLuottorajaa">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Nosta luottorajaa</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTakaisin">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>370</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Takaisin</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_kirjaudu">
     <widget class="QPushButton" name="btnKirjaudu">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>380</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Kirjaudu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPoistu">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>380</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Poistu</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cardnumber">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>100</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_pin">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>200</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>PIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>KÄYTTÄJÄTUNNUS</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelInfo">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>421</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
